packages/common-all/src/types/typesv2.ts

```diff
@@ -552,6 +531,18 @@ export type DEngineClient = Omit<DEngine, "store">;
 
 export type DStore = DCommonProps &
   DCommonMethods & {
+    /**
+     * @deprecated
```

Comment:
Do we still need to add these back?

Reply:
This is needed for `storev2` to work as `enginev2` calls store.notes. This will be removed once both are removed